### ISSUE TITLE
Fix missing images

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#17]
Missing images are images which don't load, usually because the image file has been removed or the image has the wrong web address.

Even if the image is not used, requesting a missing image will slow down the page unnecessarily.

Missing images may indicate a technical fault, and give a poor experience for the user.

Sometimes a file can exist but fail to load because of how it is used. For example, if an image is served insecurely (via HTTP) on a secure webpage (HTTPS) it will be blocked by the browser.
